captcha_solver/solver.py

```diff
@@ -1,130 +1,132 @@
-from pprint import pprint, pformat # pylint: disable=unused-import
```

Comment:
Why diff says that you removed 130 ALL lines and added 132 lines? I have no idea what is happening here.